### ISSUE TITLE
libgconf2-4 deps error when installing in Sid

### ISSUE THREAD

**Issue by groolot** [jean-emmanuel/open-stage-control#481]
In Debian, the correct new package name is `libgconf-2-4`. See https://packages.debian.org/stretch/libgconf2-4

**Comment by groolot** [comment]
```
$ sudo dpkg -i ./open-stage-control-0.41.7-amd64.deb                                                                                                                                                                               
(Reading database ... 482867 files and directories currently installed.)
Preparing to unpack .../open-stage-control-0.41.7-amd64.deb ...
Unpacking open-stage-control (0.41.7-1) over (0.41.7-1) ...
dpkg: dependency problems prevent configuration of open-stage-control:
 open-stage-control depends on libgconf2-4; however:
  Package libgconf2-4 is not installed.

dpkg: error processing package open-stage-control (--install):
 dependency problems - leaving unconfigured
Processing triggers for mime-support (3.62) ...
Processing triggers for desktop-file-utils (0.23-4) ...
Errors were encountered while processing:
 open-stage-control
```

**Comment by groolot** [comment]
This seems to be an `electron-installer-debian` bug: https://github.com/electron-userland/electron-installer-debian/blob/5895cb5f4846dfa2dc52fae53d51900238bb786e/src/dependencies.js

**Comment by jean-emmanuel** [comment]
It's just been fixed upstream: https://github.com/electron-userland/electron-installer-debian/commit/ec1dc4b2f7f9534302b4062359044f4ff2830bf3 :)

**Comment by jean-emmanuel** [comment]
Should be fixed in 0.42.0 (building/uploading right now)
